File: httpdns_android_demo/src/main/java/alibaba/httpdns_android_demo/AboutUsActivity.java
package alibaba.httpdns_android_demo;

import android.app.ActionBar;
import android.app.Activity;
import android.os.Bundle;
import android.view.Menu;
import android.view.MenuItem;

/**
 * Created by liyazhou on 17/8/29.
 */

public class AboutUsActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.demo_activity_about_us);

        // ActionBar 回退导航
        ActionBar actionBar = getActionBar();
        actionBar.setDisplayHomeAsUpEnabled(true);
    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        return true;
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        switch (item.getItemId()) {
            case android.R.id.home:
                finish();
                return true;
            default:
                return super.onOptionsItemSelected(item);
        }
    }
}

File: httpdns_android_demo/src/main/java/com/alibaba/httpdns/android/demo/setting/AboutUsActivity.kt
package com.alibaba.httpdns.android.demo.setting

import alibaba.httpdns_android_demo.R
import android.graphics.Color
import android.os.Bundle
import android.widget.ImageView
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.WindowCompat

class AboutUsActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val controller = WindowCompat.getInsetsController(window, window.decorView)
        WindowCompat.setDecorFitsSystemWindows(window, false)
        controller.isAppearanceLightStatusBars = true
        window.statusBarColor = Color.TRANSPARENT
        setContentView(R.layout.httpdns_activity_about_us)
        findViewById<ImageView>(R.id.ivBack).setOnClickListener { finish() }
    }

}